### PR TITLE
Update openssl to 1.1.1n

### PR DESCRIPTION
**Pull request by hoodmane**
Update says: 'Security Advisory: one high severity fix' so probably
a good idea to include it.

### CODE REVIEW

File: packages/ssl/meta.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ build:
     export DISTDIR=$(pwd)/dist
     cd $CPYTHONBUILD
     emcc $STDLIB_MODULE_CFLAGS -c Modules/socketmodule.c -o Modules/socketmodule.o
-    emcc $STDLIB_MODULE_CFLAGS -c Modules/_ssl.c -o Modules/_ssl.o -I$PYODIDE_ROOT/packages/openssl/build/openssl-1.1.1m/include/ \
+    emcc $STDLIB_MODULE_CFLAGS -c Modules/_ssl.c -o Modules/_ssl.o -I$(OPEN_SSL_ROOT)/include/ \
```
Comment (ryanking13) [inline packages/ssl/meta.yaml:13]:
```suggestion
    emcc $STDLIB_MODULE_CFLAGS -c Modules/_ssl.c -o Modules/_ssl.o -I$OPEN_SSL_ROOT/include/ \
```

**Review by ryanking13 — APPROVED** [review]
Thanks, @hoodmane!

File: packages/ssl/meta.yaml (hunk 1 above)
Comment (hoodmane) [reply packages/ssl/meta.yaml:13]: I was totally confused by this! It's so hard to keep track of all the different ways to substitute variables in shell and makefile and autotools etc.

Comment (ryanking13) [reply packages/ssl/meta.yaml]: The problem is that we are using `$(VAR)` syntax in substituting env variables in `cflags, cxxflags, ldflags` which means 'execute command' in a shell. We need to change this in a way compatible to shell behavior ($VAR or ${VAR})

https://github.com/pyodide/pyodide/blob/3c2ae15052fc391668d4d2bbba5ff3c3d3e910ad/pyodide-build/pyodide_build/pywasmcross.py#L526-L537

**Pull request merged**